sdk/core/azure-core/tests/test_pipeline.py

```diff
@@ -121,6 +122,13 @@ def test_requests_socket_timeout(self):
             with Pipeline(RequestsTransport(), policies=policies) as pipeline:
                 response = pipeline.run(request, connection_timeout=0.000001)
 
+    def test_retry_code_class_variables(self):
+        retry_policy = RetryPolicy()
+        assert retry_policy._RETRY_CODES is not None
+        assert len(retry_policy._RETRY_CODES) == 498
+        assert 408 in retry_policy._RETRY_CODES
+        assert 501 not in retry_policy._RETRY_CODES
```

Comment:
If you really want to lock this down:
```
assert len(retry_policy._RETRY_CODES) == 498
assert min(retry_policy._RETRY_CODES) == 408
assert max(retry_policy._RETRY_CODES) == 998
for x in [408, 500, 502, 503, 504]:
    assert x in retry_policy._RETRY_CODES
for x in range(506, 999):
    assert x in retry_policy._RETRY_CODES
```

BTW is 999 intended to be in the list? That would require e.g. `range(1000)` above (python ranges are half-open)

Reply:
you're right! Let me confirm with @xiangyan99  and @lmazuel  if 999 is included ( I would assume it must be since it's a valid code )

Reply:
confirmed with @xiangyan99 . 999 is not included.